tests/integration_tests/without_db/udfs/test_sequence_classification_single_text_udf.py

```diff
@@ -41,7 +41,11 @@ def get_dataframe(self, num_rows='all', start_col=0):
         return return_df
 
 
-@pytest.mark.parametrize("device_id", [None, 0])
+@pytest.mark.parametrize(
+    "device_id, upload_model_to_local_bucketfs", [
+        (None, model_params.base),
+        (0, model_params.base)
+    ], indirect=["upload_model_to_local_bucketfs"])
```

Comment:
and using two different fixtures wasn't and option, it probably would have easier and would look less confusing